//tensorboard/plugins/graph:keras_util_test failing at master

Can repro at cf76398963491f01ac92 with tf-nightly-20190325-py2.7.

Diff:

```
-   input: "model/func_seq_input"
+   input: "model/[u\'func_seq_input\', 0, 0, {}]"
```

Full traceback:

```
Traceback (most recent call last):
  File "/usr/local/google/home/wchargin/virtualenv/tf-nightly-20190325-py2.7/local/lib/python2.7/site-packages/absl/third_party/unittest3_backport/case.py", line 37, in testPartExecutor
    yield
  File "/usr/local/google/home/wchargin/virtualenv/tf-nightly-20190325-py2.7/local/lib/python2.7/site-packages/absl/third_party/unittest3_backport/case.py", line 162, in run
    testMethod()
  File "/usr/local/google/home/wchargin/.bazel-output-base/sandbox/linux-sandbox/26/execroot/org_tensorflow_tensorboard/bazel-out/k8-fastbuild/bin/tensorboard/plugins/graph/keras_util_test.runfiles/org_tensorflow_tensorboard/tensorboard/plugins/graph/keras_util_test.py", line 769, in test_keras_model_to_graph_def_functional_sequential_model
    self.assertGraphDefToModel(expected_proto, model)
  File "/usr/local/google/home/wchargin/.bazel-output-base/sandbox/linux-sandbox/26/execroot/org_tensorflow_tensorboard/bazel-out/k8-fastbuild/bin/tensorboard/plugins/graph/keras_util_test.runfiles/org_tensorflow_tensorboard/tensorboard/plugins/graph/keras_util_test.py", line 36, in assertGraphDefToModel
    expected_proto, keras_util.keras_model_to_graph_def(model_config))
  File "/usr/local/google/home/wchargin/virtualenv/tf-nightly-20190325-py2.7/local/lib/python2.7/site-packages/tensorflow/python/framework/test_util.py", line 1657, in assertProtoEquals
    self._AssertProtoEquals(expected_message, message, msg=msg)
  File "/usr/local/google/home/wchargin/virtualenv/tf-nightly-20190325-py2.7/local/lib/python2.7/site-packages/tensorflow/python/framework/test_util.py", line 1634, in _AssertProtoEquals
    compare.assertProtoEqual(self, a, b, normalize_numbers=True, msg=msg)
  File "/usr/local/google/home/wchargin/virtualenv/tf-nightly-20190325-py2.7/local/lib/python2.7/site-packages/tensorflow/python/util/protobuf/compare.py", line 113, in assertProtoEqual
    self.assertMultiLineEqual(a_str, b_str, msg=msg)
  File "/usr/local/google/home/wchargin/virtualenv/tf-nightly-20190325-py2.7/local/lib/python2.7/site-packages/absl/testing/absltest.py", line 979, in assertMultiLineEqual
    raise self.failureException(''.join(failure_message))
AssertionError: 
  node {
    name: "model/func_seq_input"
    attr {
      key: "dtype"
      value {
        type: DT_FLOAT
      }
    }
    attr {
      key: "keras_class"
      value {
        s: "InputLayer"
      }
    }
  }
  node {
    name: "model/sequential/dense"
-   input: "model/func_seq_input"
+   input: "model/[u\'func_seq_input\', 0, 0, {}]"
?                 ++++              +++++++++++++
    attr {
      key: "dtype"
      value {
        type: DT_FLOAT
      }
    }
    attr {
      key: "keras_class"
      value {
        s: "Dense"
      }
    }
  }
  node {
    name: "model/sequential/my_relu"
    input: "model/sequential/dense"
    attr {
      key: "dtype"
      value {
        type: DT_FLOAT
      }
    }
    attr {
      key: "keras_class"
      value {
        s: "Activation"
      }
    }
  }
  node {
    name: "model/dense_1"
    input: "model/sequential/my_relu"
    attr {
      key: "dtype"
      value {
        type: DT_FLOAT
      }
    }
    attr {
      key: "keras_class"
      value {
        s: "Dense"
      }
    }
  }
```

--- a/tensorboard/plugins/graph/keras_util.py
+++ b/tensorboard/plugins/graph/keras_util.py
@@ -156,7 +156,7 @@ def _update_dicts(name_scope,
   elif is_parent_functional_model and not is_functional_model:
     # Sequential model can take only one input. Make sure inbound to the
     # model is linked to the first layer in the Sequential model.
-    prev_node_name = _scoped_name(name_scope, inbound_nodes[0][0])
+    prev_node_name = _scoped_name(name_scope, inbound_nodes[0][0][0])
   elif not is_parent_functional_model and prev_node_name and is_functional_model:
     assert len(input_layers) == 1, (
         'Cannot have multi-input Functional model when parent model '

